test_biosignals.py

```diff
@@ -0,0 +1,35 @@
+import unittest
```

Comment:
C:\Users\Pedro\AppData\Local\Programs\Python\Python310\python.exe "C:\Program Files\JetBrains\PyCharm Community Edition 2021.2.3\plugins\python-ce\helpers\pycharm\_jb_unittest_runner.py" --path "C:/full gevie repo/Python-Shape-Game/test_biosignals.py"
Testing started at 12:12 PM ...
Traceback (most recent call last):
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2021.2.3\plugins\python-ce\helpers\pycharm\_jb_unittest_runner.py", line 35, in <module>
    sys.exit(main(argv=args, module=None, testRunner=unittestpy.TeamcityTestRunner, buffer=not JB_DISABLE_BUFFERING))
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python310\lib\unittest\main.py", line 100, in __init__
    self.parseArgs(argv)
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python310\lib\unittest\main.py", line 147, in parseArgs
    self.createTests()
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python310\lib\unittest\main.py", line 158, in createTests
    self.test = self.testLoader.loadTestsFromNames(self.testNames,
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python310\lib\unittest\loader.py", line 220, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python310\lib\unittest\loader.py", line 220, in <listcomp>
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python310\lib\unittest\loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "C:\full gevie repo\Python-Shape-Game\test_biosignals.py", line 2, in <module>
    from biosignals import Session
  File "C:\full gevie repo\Python-Shape-Game\biosignals.py", line 70, in <module>
    class Recording(Session):
  File "C:\full gevie repo\Python-Shape-Game\biosignals.py", line 75, in Recording
    fs = Session.session_data['fs']
AttributeError: type object 'Session' has no attribute 'session_data'
Launching unittests with arguments python -m unittest C:/full gevie repo/Python-Shape-Game/test_biosignals.py in C:\full gevie repo\Python-Shape-Game


Process finished with exit code 1

Empty suite

Reply:
this is the error that pops up